src/server/room-controller.spec.js

```diff
@@ -158,36 +159,133 @@ describe('room-controller', () => {
   });
 
   describe('handleGetRoomPage', () => {
-    const room = { roomId: 'roomId', name: 'Mein schöner Raum' };
+    const privateRoom = { roomId: 'roomId', name: 'Mein schöner Raum', owner: 'owner', access: ROOM_ACCESS_LEVEL.private };
+
+    beforeEach(() => {
+      clientDataMapper.mapRoom.resolves(privateRoom);
+
+      roomService.getRoomById.callsFake(roomId => {
+        if (roomId === privateRoom.roomId) {
+          return Promise.resolve(privateRoom);
+        }
+        return Promise.resolve();
+      });
+    });
+
+    describe('when the private room exists', () => {
+      describe('when the room owner calls', () => {
```

Comment:
Small hint: this would read better in the test report as
```
when the private room exists 
   and the room owner calls
   and a member calls
```

Reply:
Done, thanks for the hint.

Reply:
Cool 👍